scripts/irods/start_options.py

```diff
@@ -27,3 +27,12 @@ def add_options(parser):
                       dest='server_reconnect_flag', action='store_true', default=False,
                       help='Causes the server to attempt a reconnect after '
                       'timeout (ten minutes)')
+
+    parser.add_option('--stdout',
+                      dest='write_to_stdout', action='store_true',
+                      help='Write log messages to stdout instead of rsyslog')
+
+    parser.add_option('--test',
+                      dest='test_mode', action='store_true',
+                      help='Simultaneously writes log messages to stdout or rsyslog '
+                           'and IRODS_HOME/log/test_mode_output.log')
```

Comment:
consider rephrasing to...

'Additionally write log messages to IRODS_HOME/log/test_mode_output.log'

Reply:
Will change.

Reply:
Updated.